apps/frontend/src/app/3_organisms/FastBtcDialog/FastBtcDialog.tsx

```diff
@@ -79,6 +79,11 @@ export const FastBtcDialog: React.FC<FastBtcDialogProps> = ({
     [isMobile],
   );
 
+  // Reset the selected index to 0 in case the connected account changes because the new account may have the Send tab hidden and it would crash the app if we tried to access that index
```

Comment:
If possible, can we also close the Funding dialog when last wallet is disconnected? Current behaviour is that dialog remains open in this state (not a big deal since you can't proceed forward, but preferably we close it fully):
![image](https://user-images.githubusercontent.com/765825/221850457-474ca2d6-70f4-4d80-88e7-b8176e3d91ac.png)

Reply:
Good catch, I just pushed the fix :)